src/components/shared/StackSettings.js

```diff
@@ -160,7 +160,9 @@ class StackSettingsImpl extends PureComponent<Props> {
           {hasAllocations ? (
             <li className="stackSettingsListItem stackSettingsFilter">
               <label>
-                <Localized id="StackSettings--summarize">Summarize:</Localized>{' '}
+                <Localized id="StackSettings--display-base-data">
+                  Use underlying data:
```

Comment:
nitpick: This label is different than what's in the ftl file.

Thought (no action needed) (wearing my internationalization hat): I'm somewhat skeptical of the benefits of duplicating the label text in the React components even if that's what fluent-react is recommending. It seems a bit confusing when you get inevitable label mismatches. The localization id should be good enough for localizers to understand the use, so I would think it should be good enough for developers to understand the content.

Reply:
Yeah I think the same now :-) Initially I wasn't sure, but now that tests were hooked to load the l10n information as well, and we're waiting for the translation before rendering at load time, I believe we're all good.